packages/components/src/internal/components/entities/EntityInsertPanel.tsx

```diff
@@ -1316,11 +1315,10 @@ export const EntityInsertPanel: FC<{ location?: Location } & OwnProps> = memo(pr
             selectionKey,
             tab: parseInt(tab, 10),
             target,
-            user,
         };
     }, [location]);
 
-    return <EntityInsertPanelFormSteps {...entityInsertPanelProps} {...fromLocationProps} />;
+    return <EntityInsertPanelFormSteps {...entityInsertPanelProps} {...fromLocationProps} user={user} />;
```

Comment:
Seems unrelated, but potentially important.

Reply:
As mentioned in standup, this is just a minor cleanup that I noticed after fixing the test case in platform.